camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlInputFormat.java

```diff
@@ -135,16 +135,21 @@ public static void setLogger(Logger log) {
     return topicMetadataList;
   }
 
-  private SimpleConsumer createConsumer(JobContext context, String broker) {
+  private SimpleConsumer createBrokerConsumer(JobContext context, String broker) {
```

Comment:
1. Why not name it createSimpleConsumer?
2. This method (as well as lots of other methods) doesn't access object state, and can be static. But this may be too much change and not feasible.